diesel_derives/src/as_changeset.rs

```diff
@@ -20,22 +20,28 @@ pub fn derive(item: syn::DeriveInput) -> Result<quote::Tokens, Diagnostic> {
     let struct_name = &model.name;
     let table_name = model.table_name();
 
-    let (_, ty_generics, where_clause) = item.generics.split_for_impl();
-    let mut impl_generics = item.generics.clone();
-    impl_generics.params.push(parse_quote!('update));
-    let (impl_generics, _, _) = impl_generics.split_for_impl();
+    let (impl_generics, ty_generics, where_clause) = item.generics.split_for_impl();
+    let mut ref_generics = item.generics.clone();
+    ref_generics.params.push(parse_quote!('update));
```

Comment:
Do we actually need to split these? Rust doesn't mind if there's an unused lifetime in the generics.